src/lib/accounts/AccountSuperhero.ts

```diff
@@ -36,11 +36,11 @@ export class AccountSuperhero extends AccountBase {
 
   signTransaction(txBase64: Encoded.Transaction, options: any): Promise<Encoded.Transaction> {
     if (IS_MOBILE_APP && options.aeppOrigin) {
-      return this.fgPermissionCheckAndSign(POPUP_TYPE_SIGN, txBase64, options, options.aeppOrigin);
+      return this.fgPermissionCheckAndSign(METHODS.sign, txBase64, options, options.aeppOrigin);
     }
     if (IS_EXTENSION_BACKGROUND) {
       return this.bgPermissionCheckAndSign(
-        POPUP_TYPE_TX_SIGN,
+        METHODS.sign,
```

Comment:
`POPUP_TYPE_SIGN` and `POPUP_TYPE_TX_SIGN` values are different, You can use `METHODS.sign` at both places but need to migration permission values of the key('sign' -> 'transaction.sign')

Reply:
Yes, that's true - the values are different. And this actually was wrong. Let's break it to small pieces: The `POPUP_TYPE_TX_SIGN` was equal to `METHODS.sign` which is equal to `"transaction.sign"` string. The `POPUP_TYPE_SIGN` was equal to `"sign"` string.
In our previous code (and the new one) there was no logic for method with value of `"sign"` - this method: https://github.com/aeternity/superhero-wallet/blob/develop/src/store/modules/permissions.js#L75 was always returning `undefined` in this case.